bftengine/src/bftengine/CollectorOfThresholdSignatures.hpp

```diff
@@ -364,12 +364,12 @@ class CollectorOfThresholdSignatures {
 
         acc->setExpectedDigest(reinterpret_cast<unsigned char*>(expectedDigest.content()), DIGEST_SIZE);
 
-        acc->getFullSignedData(bufferForSigComputations, bufferSize);
+        acc->getFullSignedData(const_cast<char*>(bufferForSigComputations.data()), bufferSize);
```

Comment:
If you remove the const on line 354, you won't need this cast.

Reply:
Fixed.